src/core/Akka.Streams/Util/ObjectExtensions.cs

```diff
@@ -21,5 +21,10 @@ public static class ObjectExtensions
         /// <param name="obj">TBD</param>
         /// <returns>TBD</returns>
         public static bool IsDefaultForType<T>(this T obj) => EqualityComparer<T>.Default.Equals(obj, default(T));
+        
+        /// <summary>
+        /// Wraps object to the <see cref="Option{T}"/> monade
+        /// </summary>
+        public static Option<T> AsOption<T>(this T obj) => new Option<T>(obj);
```

Comment:
This is something I am really missing very often. But maybe there also exist some method like that? Did not find simple way to convert object to optional value - `new Option(obj)` requires specifying target type.